boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm33.yaml

```diff
@@ -19,4 +19,6 @@ supported:
   - uart
   - i2c
   - counter
+  - dma
+  - spi
```

Comment:
changing board yaml before commit that enables the testing overlays? is it intended ? (maybe it doesnt matter since it never was enabled before this)

Reply:
Squash the board and test modification related to spi_edma commit into one commit.